.github/workflows/fetch_filter_resources.yaml

```diff
@@ -0,0 +1,75 @@
+name: Weekly resource fetching and community filtering
+
+on:
+  workflow_dispatch:
+  schedule:
+    #Every Sunday at 8:00 am
+    - cron: "0 8 * * 0" 
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "tools"
+  cancel-in-progress: false
+
+jobs:
+  fetch-filter:
+    runs-on: ubuntu-20.04
+    name: Fetch tool stepwise and merge, fetch tutorials and filter the resources for communities
+    strategy:
+      max-parallel: 1 #need to run one after another, since otherwise there is a chance, that mulitple jobs want to push to the results branch at the same time (which fails due to merge)
+      matrix:
```

Comment:
This combined WF triggers a subset job, where each repositories*.list is processed individually. The way it's now combined, everything after the merge part does not really make sense ... and the  `Fetch list of all available servers` should probably also not be a part of the matrix subset. I would propose to run the `Fetch all tool stepwise` in an individual job and store the results as an artifact. Then use the artifact as input for the merge part and have that as another job.

Reply:
I adapted the CRON workflow to have 3 jobs
1. For getting the available servers and storing the TSV as artifact
2. For triggering the subset jobs to fetch tools in parallel and storing the outputs as artifact
3. For merging the TSV and continuing afterwards